aidants_connect_web/views/mandat.py

```diff
@@ -240,6 +219,32 @@ def attestation_projet(request):
 @login_required
 @activity_required
 def attestation_final(request):
+    return __attestation_visulisation(request, settings.MANDAT_TEMPLATE_PATH)
```

Comment:
Il manquerait pas une lettre dans le nom de la fonction ?
```suggestion
    return __attestation_visualisation(request, settings.MANDAT_TEMPLATE_PATH)
```